packages/config/src/handlers/forms/validation.ts

```diff
@@ -300,6 +312,9 @@ const OPTIONAL_FIELDS_IN_SECTION: Record<string, string[] | undefined> = {
     'iD',
```

Comment:
@euanmillar Do we want to keep the 'iD' field here as an optional field for backwards compatibility or have the implementers change it to `{sectionName}NationalId` as a manual migration step